src/ts/components/extensions/richtexteditor/fragments/RichTextEditor.tsx

```diff
@@ -28,13 +28,28 @@ import {
     LineHeight,
 } from '@tiptap/extension-text-style';
 import Image from '@tiptap/extension-image';
+import CodeBlockLowlight from '@tiptap/extension-code-block-lowlight';
+import { createLowlight } from 'lowlight';
+import ts from 'highlight.js/lib/languages/typescript';
+import js from 'highlight.js/lib/languages/javascript';
+import python from 'highlight.js/lib/languages/python';
+import css from 'highlight.js/lib/languages/css';
+import plaintext from 'highlight.js/lib/languages/plaintext';
+
 import {
     getLoadingState,
     setPersistence,
     newRenderDashComponent,
     getContextPath,
 } from '../../../../utils/dash3';
 
+const lowlight = createLowlight();
+lowlight.register({ ts });
+lowlight.register('js', js);
+lowlight.register('python', python);
+lowlight.register('css', css);
+lowlight.register('text', plaintext);
```

Comment:
If I'm understanding this right, can't this be written:
```suggestion
lowlight.register({ ts, js, python, css, plaintext });
```
?

Also I wonder if you want to include bash and/or shell, for documenting CLI usage?